File: python_draft/src/java/type3/FindMaxOriginal.java
public class FindMaxOriginal {
public static int findMax(int[] values) {
    int maxVal = values[0];
    for (int i=1;i<values.length;i++) {
        if (values[i] > maxVal) maxVal = values[i];
    }
    return maxVal;
}

}

File: python_draft/src/cpp/type3/find_max_original.cpp
#include <vector>
int findMax(const std::vector<int>& values){
    int maxVal = values[0];
    for(size_t i=1;i<values.size();++i){
        if(values[i] > maxVal) maxVal = values[i];
    }
    return maxVal;
}
